### PR TITLE
Kh/movable ink beta

### PR DESCRIPTION
**Pull request by KellieHawks**
# Pull Request/Issue Resolution

**Description of Change:**
> I'm changing..... (could be a link, a new image, a new section, etc.)... because...

Closes #**ISSUE_NUMBER_HERE**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [ ] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>

- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @KellieHawks as a reviewer when the your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

**Comment by KellieHawks** [comment]
@cla-bot check

File: _docs/_hidden/private_betas/movable_ink.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+&#45; In-App Messages and Content Cards: Make sure that the image URL will be unique for each impression. This can be done by appending the appropriate liquid so that each URL is different. See [In-App and Content Card Messages Instructions][Instructions]. 
+<br><br>
+__Having trouble showing GIFs on Android?__<br>
+&#45; Android requires [GIF support][GIFsupport] in implementation. If you do not have this setup, follow the instructions to do so [here][GIFsupport].
+
+[1]: https://movableink.com/
+[datasource]: ({% image_buster /assets/img/movable_ink/movable_ink1.png %})
+[support]: https://support.movableink.com/
+[GIFsupport]: {{ site.baseurl }}/developer_guide/platform_integration_guides/android/in-app_messaging/customization/#gifs-IAMs
+[Instructions]: {{ site.baseurl }}/partners/channel_extensions/creative_and_personalization/intelligent_creative/movable_ink/#in-app-and-content-card-messages
```
Comment (zzhaobraze) [inline _docs/_hidden/private_betas/movable_ink.md]: This link is broken

Comment (KellieHawks) [reply _docs/_hidden/private_betas/movable_ink.md]: fixed

File: _docs/_hidden/private_betas/movable_ink.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+|| Optimize | ✗ | ✔* | * Must Use Branch's Deeplinking solution |
+| Targeting Rules | Date | ✔* | ✔ | * Supported but not recommended because push notifications are cached upon receipt and do not refresh |
+|| Day of Week | ✔* | ✔ | * Supported but not recommended because push notifications are cached upon receipt and do not refresh |
+|| Time of Day | ✔* | ✔ | * Supported but not recommended because push notifications are cached upon receipt and do not refresh |
+| Stories/Behavior Activity | | ✔* | ✔* | * The unique user identifier used for Braze must be linked to your ESP’s identifier |
+| Deep-Linking within the app | | ✔* | ✔* | * Must use Branch’s deep linking solution |
+| Apps | Countdown Timer | ✔* | ✔ | * Supported but not recommended because push notifications are cached upon receipt and do not refresh |
+|| Polling | ✗ | ✔* | * After voting, will leave the app to be a mobile landing page |
+|| Scratch Off | ✔* | ✔* | * On click, will leave the app for the Scratch Off experience |
+|| Video | ✔* | ✔* | * Animated GIFs only, <br>For Android, Braze requires [GIF support][GIFsupport] in implementation |
```
Comment (zzhaobraze) [inline _docs/_hidden/private_betas/movable_ink.md]: This might be a good usage of the `reset-td-br-1` and `reset-td-br-1` feature
moveableinktable

Comment (KellieHawks) [reply _docs/_hidden/private_betas/movable_ink.md]: Thank you!

**Pull request merged**